package.json

```diff
@@ -72,7 +72,12 @@
           },
           "dbt.enableNewDocsPanel": {
             "type": "boolean",
-            "description": "Enable the new docs panel in dbt."
+            "description": "Enable the new docs panel in dbt.",
+            "default": true
+          },
+          "dbt.enableTests": {
+            "type": "boolean",
+            "description": "Enable the new tests experience in documentation editor."
```

Comment:
I don't think this is required as we already have a button to show/display tests.

Reply:
In the first release, we dont want to show tests by default. Once we introduce infer tests from dbt, we will enable it by default